utils_cv/action_recognition/model.py

```diff
@@ -273,7 +273,7 @@ def fit(
                 self.save(
                     os.path.join(
                         model_dir,
-                        "{model_name}_{self.epoch}.pt".format(
+                        "{model_name}_{epoch}.pt".format(
```

Comment:
Should the line be as suggested:
"{model_name}_{epoch}.

Or should it rather be like below ("epoch" does not seem to be defined)
"{model_name}_{e}

Reply:
Actually `epoch` is defined by the parameter of `format`. See L277. This fix works perfectly for me.

Reply:
Thanks, I see it now!